src/baseCanvas/Canvas.tsx

```diff
@@ -158,14 +159,15 @@ export class BaseCanvas extends Component<Props> {
         width: "100%",
         height: this.props.canvasPositionAndSize.height,
         cursor: this.props.cursor || "pointer",
-        //   border: "1px solid gray",
-        top: this.props.canvasPositionAndSize.top,
         left: this.props.canvasPositionAndSize.left,
+        bottom: "0px",
         position: "absolute",
+        right: "250px",
+        marginTop: "30px",
       }}
     >
       <canvas
-        style={{ pointerEvents: "inherit" }}
+        style={{ pointerEvents: "inherit", zIndex: 100 }}
```

Comment:
In my experience we should be cautious with zIndex - can you justify for us why this was needed and what alternatives you considered?